docs/src/pages/components/Icon.vue

```diff
@@ -47,6 +47,12 @@
                 <md-table-cell><code>String</code></md-table-cell>
                 <md-table-cell>The src of the image file. Can be SVG icons too!</md-table-cell>
               </md-table-row>
+              <md-table-row>
+                <md-table-cell>md-iconset</md-table-cell>
+                <md-table-cell><code>String</code></md-table-cell>
+                <md-table-cell>The icon set font you'll use. For example, in 
```

Comment:
This this the tricky part. For some reason it's not working, and that's why i tried a boolean one. However i'll make it string again, and see what happens. Maybe it was a problem between the couch and the laptop, ;-)